src/java_tools/buildjar/java/com/google/devtools/build/buildjar/BazelScannerSuppliers.java

```diff
@@ -19,132 +19,8 @@
-import com.google.errorprone.bugpatterns.NonCanonicalStaticImport;
-import com.google.errorprone.bugpatterns.NonFinalCompileTimeConstant;
-import com.google.errorprone.bugpatterns.NonRuntimeAnnotation;
-import com.google.errorprone.bugpatterns.NullTernary;
-import com.google.errorprone.bugpatterns.NullableOnContainingClass;
-import com.google.errorprone.bugpatterns.OptionalEquality;
-import com.google.errorprone.bugpatterns.OptionalMapUnusedValue;
-import com.google.errorprone.bugpatterns.OptionalOfRedundantMethod;
-import com.google.errorprone.bugpatterns.PackageInfo;
-import com.google.errorprone.bugpatterns.ParametersButNotParameterized;
-import com.google.errorprone.bugpatterns.PreconditionsInvalidPlaceholder;
-import com.google.errorprone.bugpatterns.PrivateSecurityContractProtoAccess;
-import com.google.errorprone.bugpatterns.ProtoBuilderReturnValueIgnored;
-import com.google.errorprone.bugpatterns.ProtoStringFieldReferenceEquality;
-import com.google.errorprone.bugpatterns.ProtoTruthMixedDescriptors;
-import com.google.errorprone.bugpatterns.ProtocolBufferOrdinal;
-import com.google.errorprone.bugpatterns.RandomCast;
-import com.google.errorprone.bugpatterns.RandomModInteger;
-import com.google.errorprone.bugpatterns.RequiredModifiersChecker;
-import com.google.errorprone.bugpatterns.RestrictedApiChecker;
-import com.google.errorprone.bugpatterns.ReturnValueIgnored;
-import com.google.errorprone.bugpatterns.SelfAssignment;
-import com.google.errorprone.bugpatterns.SelfComparison;
-import com.google.errorprone.bugpatterns.SelfEquals;
-import com.google.errorprone.bugpatterns.ShouldHaveEvenArgs;
-import com.google.errorprone.bugpatterns.SizeGreaterThanOrEqualsZero;
-import com.google.errorprone.bugpatterns.StreamToString;
-import com.google.errorprone.bugpatterns.StringBuilderInitWithChar;
-import com.google.errorprone.bugpatterns.SubstringOfZero;
-import com.google.errorprone.bugpatterns.SuppressWarningsDeprecated;
-import com.google.errorprone.bugpatterns.TestParametersNotInitialized;
-import com.google.errorprone.bugpatterns.TheoryButNoTheories;
-import com.google.errorprone.bugpatterns.ThrowIfUncheckedKnownChecked;
-import com.google.errorprone.bugpatterns.ThrowNull;
-import com.google.errorprone.bugpatterns.TreeToString;
-import com.google.errorprone.bugpatterns.TruthSelfEquals;
-import com.google.errorprone.bugpatterns.TryFailThrowable;
-import com.google.errorprone.bugpatterns.TypeParameterQualifier;
-import com.google.errorprone.bugpatterns.UnicodeDirectionalityCharacters;
-import com.google.errorprone.bugpatterns.UnicodeInCode;
-import com.google.errorprone.bugpatterns.UnnecessaryTypeArgument;
-import com.google.errorprone.bugpatterns.UnusedAnonymousClass;
-import com.google.errorprone.bugpatterns.UnusedCollectionModifiedInPlace;
-import com.google.errorprone.bugpatterns.VarTypeName;
-import com.google.errorprone.bugpatterns.WrongOneof;
-import com.google.errorprone.bugpatterns.XorPower;
+import com.google.errorprone.bugpatterns.*;
```

Comment:
Wildcard imports are not discouraged internally, this _might_ cause submit failures on import.

Reply:
Ah, sorry, I didn't mean to change this. I will revert these.

Reply:
I'm belatedly noticing this is for `release-7.6.0` and will only be on github, so should not an issue. But keeping the changes minimal is a good idea, to avoid potential future merge conflicts. Thanks!